lua/lspconfig/volarvuels.lua

```diff
@@ -0,0 +1,88 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'volarvuels'
+
+-- TODO: This is the name if you download volar-server from the AUR
+-- need to wait for https://github.com/johnsoncodehk/volar/issues/458
+local bin_name = 'volar-server'
+
```

Comment:
This comes from https://aur.archlinux.org/packages/volar-server/ . The npm package @volar/server doesn't expose a binary, so you need to refer to it by `node $(npm get prefix)/lib/node_modules/@volar/server/out/index.js`.

I guess it's better to wait for the upstream volar issue 458 to be implemented so we don't have to implement a wrapper script for getting the path.

Reply:
This is now fixed & closed. The executable name is vue-server. This change is only published in an alpha version so it still may be worth waiting a little bit. I'll change bin-name to `vue-server` in a second.